### ISSUE TITLE
kodi low framerate

### ISSUE THREAD

**Issue by Alistair1231** [m1k1o/neko-apps#10]
Hi, I jsut got kodi running after making some changes to the Dockerfile in order to get the official repos to work and use a volume for the userdata directory.
Everything works really well, however even after increasing shm_size to 3 and later 4 gb (which did help), I still have micro stutters in my playback. My current theory is, that hardware decoding doesn't work. Is that possible? How can I activate it?   
`NEKO_HWENC: VAAPI` makes it so I cannot kodi itself, and I am very lost regarding `NEKO_VIDEO` and gstreamer.

**Comment by m1k1o** [comment]
`NEKO_VIDEO` and gstreamer should be irrelevant, if other programs in neko work fine.

Then its about decoding and kodi itself. Its new for me, only experimental. Maybe kodi can be set up in a way to use GPU to make decoding better.

**Comment by m1k1o** [comment]
> Isn't NEKO_HWENC exactly about that?

Yes, but that is hw encoding for neko itself. Kodi needs to use it and that is separate program. Neko runs just finde without hw encodoing from my experinece.

**Comment by m1k1o** [comment]
https://github.com/m1k1o/neko-apps/pull/14
